test/test_testing.py

```diff
@@ -5083,12 +5243,16 @@ def test_exec_service_context(self):
             "group-id": 2,
             "environment": {"foo": "bar", "foobar": "barfoo"}
         }
-        self.container.add_layer(label="test", layer={
-            "summary": "", "description": "", "services": {"test": service}
-        })
-        args_history = []
-
-        def handler(args):
+        layer = ops.pebble.Layer(
+            ops.pebble.LayerDict(
```

Comment:
I think we should revert this one to a plain dict, which is expected and valid usage. The `LayerDict` is really more for type checking the fields, not instantiation.